app/scripts/constants.js

```diff
@@ -85,7 +85,7 @@ window.OPENSHIFT_CONSTANTS = {
   ],
 
   ENABLE_TECH_PREVIEW_FEATURE: {
-    pipelines: true
+    catalog_landing_page: false
```

Comment:
maybe expanding to service_catalog_landing_page OR just have some extra vague like new_landing_page_experience